boat-quay/boat-quay-rules/src/main/kotlin/com/backbase/oss/boat/quay/ruleset/PluralizeResourceNamesRule.kt

```diff
@@ -12,7 +12,7 @@ import org.zalando.zally.rule.api.*
 @Rule(
         ruleSet = BoatRuleSet::class,
         id = "B010",
-        severity = Severity.HINT,
+        severity = Severity.MUST,
```

Comment:
This change seems to be outside of the scope of this PR and together with the change in https://github.com/Backbase/backbase-openapi-tools/pull/779/files#diff-0d37ac6bc75b575f896b44ed22963ae103450faad888a8dd3ce346bcafd8af16R59 forces teams to fix *now* which not possible because changing the resource name is a breaking change.

Reply:
In the scope of this change, I made the same behavior. According to old SEVERITY, this rule was a HINT, but our LintMojo didn't check for severity levels, so any rule mismatch requires a fix. And it's not strict for a new Rule B014(Examples check).

Reply:
Sorry, I think the github UI was playing tricks on me... it looked to me like this change was on another rule.

Reply:
agree, about reading logs until fail.
I will improve the error by adding a missed property.
Additionally, should we check only 2xx or others as well?

Reply:
Most other response codes will be 4xx and 5xx (by the way: our guideline says to not define the 5xx because infra may throw these at the client anyway. Without the expected Error object...) but if you add this rule, then it should be for all examples. Can you also cover the POST/PUT request payloads?

Reply:
done